src/pages/index.module.scss

```
@@ -49,6 +50,21 @@ $color-background-stages-seconde: #51BCB3;
   }
 }
 
+.worldSkillsBanner {
```

```suggestion
.hero.worldSkills {
```
C'est un sélecteur un peu plus robuste : c'est un hero, mais c'est pas n'importe lequel, c'est celui pour les WorldSkills